datumaro/plugins/voc_format/converter.py

```diff
@@ -283,7 +283,10 @@ def _export_annotations(self, item: DatasetItem, *, image_filename: str, lists:
                 size_elem = ET.SubElement(root_elem, "size")
                 ET.SubElement(size_elem, "width").text = str(w)
                 ET.SubElement(size_elem, "height").text = str(h)
-                ET.SubElement(size_elem, "depth").text = ""
+                depth = ""
+                if item.media.data is not None:
+                    depth = str(item.media.data.shape[-1])
```

Comment:
In many cases VOC export is going to take much more time now, because the images have to be loaded.

Reply:
The images with coco or other formats are not loaded, so item.media.data is None.
It is run when there are the loaded images. If not, it keeps "".

Any suggestion to set depth?

Reply:
`item.media.data` is None only when the image file is not available, but we have metainfo. If you wanted to avoid extra loading, you need to use [has_data](https://github.com/openvinotoolkit/datumaro/blob/develop/datumaro/components/media.py#L113)

Reply:
I see. Then I will update as using has_data you suggested soon. Do you think it should be included in v0.3.1?

Reply:
I can't answer on this question, release logic is out of my responsibility now.